autoload/neomake/makers/ft/rst.vim

```diff
@@ -23,3 +23,19 @@ function! neomake#makers#ft#rst#rstcheck()
             \ '%E%f:%l: (SEVERE/4) %m',
         \ }
 endfunction
+
+" TODO: determine proper path of current .rst file upwards in sear for conf.py file, use that instead of "docs"
+" TODO: ask noomake for a temporary folder instead of /tmp/neomake-sphinx
```

Comment:
You can use Vim's `tempfile()` and then `mkdir` on it.
Not sure if a function in utils makes sense for this?!
The directory should be based on the current project, right? (for the `outdir/.doctrees` cache)
Could we use `/dev/null` for the `outdir`, but specify `-d` for the cache?